File: hun/prog2/java/penteki_gyakorlatok/week_01/metodusok2/Main.java
class Main
{
    public static void main(String[] args)
    {
        int x = 7;

        if (paros(x)) {
            System.out.println("páros");
        }
        else {
            System.out.println("páratlan");
        }

        a();
    }

    static void b() {
        System.out.println("hello from b()");
    }

    static void a() {
        b();
    }

    static boolean paros(int n)
    {
        if (n % 2 == 0) {
            return true;
        }
        // else
        return false;
    }
}

File: hun/prog2/java/penteki_gyakorlatok/week_01/metodusok2/main.c
#include <stdio.h>

int paros(int n)
{
    if (n % 2 == 0) {
        return 1;
    }
    // else
    return 0;
}

void b() {
    puts("hello from b()");
}

void a() {
    b();
}

int main()
{
    int x = 6;

    if (paros(x)) {
        printf("páros\n");
    }
    else {
        printf("páratlan\n");
    }

    a();

    return 0;
}
